src/EditorFeatures/CSharpTest/Diagnostics/GenerateType/GenerateTypeTests.cs

```diff
@@ -444,6 +444,25 @@ private class Goo
 index: 2);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateType)]
+        public async Task TestGenerateClassFromNullableFieldDeclarationIntoSameType()
+        {
+            await TestInRegularAndScriptAsync(
+@"class Class
```

Comment:
does this need `#nullable enable` ?

Reply:
Maybe....? I wonder if it works because the compiler may choose to annotate the symbol regardless of the mode. But that's of course crazy depending on that. :-)

Reply:
my guess is that the nullability here doesn't actually matter :) The type is the same type, whether this field can be null or not.

Reply:
Right....so yes, it needs a `#nullable enable` if we want to have any hope this is doing what I think it's doing.